core/testcasecontroller/testcase/testcase.py

```diff
@@ -105,8 +105,8 @@ def compute_metrics(self, paradigm_result, dataset, **kwargs):
 
         test_dataset_file = dataset.test_url
         test_dataset = dataset.load_data(test_dataset_file,
-                                         data_type="eval overall",
-                                         label=dataset.label)
+                                         data_type="eval overall")
+                                        #  label=dataset.label)
```

Comment:
Why comment this out?

Reply:
have modified this comment back to the default one. check the new commit plz